pages/builds/parallelizing_builds.md.erb

```diff
@@ -36,11 +36,11 @@ To run the same step over all 5 of the agents we can set the parallelism field f
 
```

Comment:
Ah yeah. So why is that a thing people want to do? Why is it good to run the same tests at the same time on different things?